content/03.methods.md

```diff
@@ -11,10 +11,50 @@
 2. Describe metadata statistics on PubMed central (number of articles, how many articles were processed
 
 ### Comparing Corpora
-1. Spacy to process text via - Lemmatization, removal of stop words
-2. Describe counting frequencies of each lemma
-3. Describe using chi-square test
-4. Describe how to calculate the likelihood and log odds ratio
+The first step in linguistic analysis is to preprocess the text for ease of computation.
+This step consists of removing non-informative words (i.e., stop words) as well as converting words into their root form (i.e., lemmatization).
+Throughout our analysis we encountered symbols that aren't considered words (e.g., $\pm$), so we will refer to words and symbols as tokens to avoid confusion.
+We used spaCy [@raw:honnibal2017spacy] to perform the above preprocessing tasks on the bioRxiv and PubMed Central corpora.
+
+Following the cleaning process, we calculated the frequency of each token between both corpora.
+Due to the large influx of unique tokens, we subsampled our analysis to use the union of the top 100 most frequent tokens between both corpora.
+For each token we generated a contingency table as described in [@doi:10.1075/ijcl.6.1.05kil] (Table {@tbl:term-freq-contingency-table}).

```

Comment:
I'm not sure that you need a description of how to make a contingency table in this research paper, though this text could be good for a review (maybe in the future but not now 😁 ). What do you think about removing the table and simplifying this to 
```suggestion
For each token we generated a contingency table and calculated the odds ratio for each of these tokens.
```

Then this would be joined with the portion below.